src/app/pages/services/courses.service.ts

```diff
@@ -0,0 +1,89 @@
+import { Injectable } from '@angular/core';
```

Comment:
Jak już wspomniałem w innym komentarzu z logicznego punktu widzenia w folderze `pages` nie powinny znajdować się "serwisy", za to mogłyby trafić do `sharedModule` ;)